x/ibc-address-limiter/contracts/address-limiter/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "address-limiter"
+version = "0.1.0"
+authors = ["SerigneCheikh98 <fallserignecheikh.98@gmail.com>"]
+edition = "2021"
+
+exclude = [
+  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+  "contract.wasm",
+  "hash.txt",
+]
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# for more explicit tests, cargo test --features=backtraces
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+# Use the verbose responses feature if you want to include information about
+# the remaining quotas in the SendPacket/RecvPacket responses
+verbose_responses = []
+
+[package.metadata.scripts]
+optimize = """docker run --rm -v "$(pwd)":/code \
+  --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+  cosmwasm/rust-optimizer:0.12.6
+"""
+
+[dependencies]
```

Comment:
Check the versions of dependencies: there should be new releases of basic contracts.